next/src/utils/get-legal-links.ts

```diff
@@ -0,0 +1,22 @@
+import sanityFetch from './sanity.fetch';
+
+type getLegalLinkSTypes = {
+  privacyPolicy: string;
+  termsAndConditions: string;
+};
+
+export default async function getLegalLinkS(): Promise<getLegalLinkSTypes> {
+  return await query();
+}
+
+async function query(): Promise<getLegalLinkSTypes> {
+  return await sanityFetch({
+    query: /* groq */ `
+      *[_id == 'global'][0] {
+        privacyPolicy,
+        termsAndConditions,
+      }
+    `,
+    tags: ['global'],
+  });
+}
```

Comment:
Isn't it will be better to put the `query` code straight into the `getLegalLinks` function?

Reply:
Sure it will.